action.yml

```diff
@@ -0,0 +1,130 @@
+name: Generate ISO
+description: Creates an ISO for installing an OSTree container
+
+inputs:
+  ARCH:
+    description: Architecture for image to build
+    required: true
+    default: x86_64
+  IMAGE_NAME:
+    description: Name of the source container image
+    required: true
+    default: base-main
+  IMAGE_REPO:
+    description: Repository containing the source container image
+    required: true
+    default: ghcr.io/ublue-os
+  VARIANT:
+    description: "Source container variant. Available options can be found by running `dnf provides system-release`. Variant will be the third item in the package name. Example: `fedora-release-kinoite-39-34.noarch` will be kinonite"
+    required: true
+    default: Kinoite
+  VERSION:
+    description: Fedora version of installer to build
+    required: true
+    default: "39"
+  IMAGE_TAG:
+    description: Tag of the source container image
+    required: false
+  WEB_UI:
+    description: Enable Anaconda WebUI
+    required: true
+    default: "false"
+  ACTION_REPO:
+    description: Repository with the build action
+    required: false
+    default: ${{ github.repository }}
+  ACTION_REF:
+    description: Repository ref for the build action
+    required: false
+    default: ${{ github.ref }}
+
+runs:
+  using: composite
+  steps:
+    - name: Cleanup host
```

Comment:
We've got an action for this

https://github.com/ublue-os/remove-unwanted-software